### DISCUSSION TITLE
pywinstyles.apply_style doesnt work properly on CTkinter

### DISCUSSION THREAD

**Discussion by eaglecode0** [Akascape/py-window-styles#11] [Q&A]
`import customtkinter
import pywinstyles

root = customtkinter.CTk()

pywinstyles.apply_style(root, style="aero")

root.mainloop()`

or 

`import customtkinter
import pywinstyles

root = customtkinter.CTk()

pywinstyles.apply_style(root, "aero")

root.mainloop()`

Does not work, and there is no proper examples.
I tried using regular tk and it works fine, why is the problem whit CTk? it just remains in the CTk dark theme

**Comment by Akascape** [comment]
@eaglecode0 I guess you have to turn on the `transparency effects` mode in settings.

![](https://www.groovypost.com/wp-content/uploads/2023/06/taskbar-translucent-win11-transparency-on.png)
